app/deliver_grant_funding/forms.py

```diff
@@ -223,6 +231,7 @@ def __init__(self, *args: Any, question_type: QuestionDataType, **kwargs: Any) -
                 DataRequired("Enter the options for your list"),
                 _validate_no_blank_lines,
                 _validate_no_duplicates,
+                _validate_max_list_length(max_length=100),
```

Comment:
Should this use `config.MAX_DATA_SOURCE_ITEMS`?

Reply:
done